python/dgl/nn/pytorch/factory.py

```diff
@@ -222,14 +232,22 @@ def forward(self, x, segs, algorithm='bruteforce-blas', dist='euclidean'):
               :math:`\sqrt{\sum_{i} (x_{i} - y_{i})^{2}}`.
             * 'cosine': Use cosine distance.
             (default: 'euclidean')
+        exclude_self : bool, optional
+            If True, the output graph will not contain self loop edges, and each node will not
+            be counted as one of its own k neighbors.  If False, the output graph will contain
+            self loop edges, and a node will be counted as one of its own k neighbors.
+        output_batch : bool, optional
+            If True, the output graph will have batch information set according to ``segs``.
+            If False, the output graph will not have batch information set.
```

Comment:
Ditto.

Reply:
Updated.  Further discussion is on the first one.